include/picongpu/plugins/Emittance.hpp

```diff
@@ -869,35 +851,22 @@ namespace picongpu
                 0,
                 commGather
             ));
-            if (gatherRank ==0){
-                std::cout << "y_offsets if gatherRank = 0: ";
-                for (int i = 0; i < gatherSize; i++){
-                    std::cout << y_offsets.at(i) << " ";
-                }
-                std::cout << std::endl;
-            }
-            if (gatherRank ==0){
-                std::cout << "globalCount_e.getDataPointer( ) if gatherRank = 0: ";
-                for (int i = 0; i < subGrid.getGlobalDomain().size.y(); i++){
-                    std::cout << globalCount_e.getDataPointer( )[i] << " ";
-                }
-                std::cout << std::endl;
-            }
+
             /* print timestep, emittance to file: */
             if( writeToFile )
             {
                 using dbl = std::numeric_limits< float_64 >;
                 outFile.precision( dbl::digits10 );
-                if (currentStep > 0){
+                if (currentStep > 0.0){
                     int startWindow_y = movingWindow.globalDimensions.offset.y();
                     int endWindow_y = movingWindow.globalDimensions.size.y() + startWindow_y;
                     if (fisttimestep == true) 
                     {
-                        outFile << "#step 0 " ; // #step, emit_all
-                        for (int i = startWindow_y; i < endWindow_y; i+=10)
+                        outFile << "#step emit_all " << std::scientific;
+                        for (int i = startWindow_y; i < (endWindow_y + 10); i+=10)
```

Comment:
we should expose the magic number `10` in a follow up to the users as plugin option.

Reply:
offline discussed: This can only be removed if the plain data of `px^2` ,... is written to the output.